IoTBlockly/ChakraHost/ChakraHostAsync.cs

```diff
@@ -51,7 +56,7 @@ public string init()
             // ES6 Promise callback
             JavaScriptPromiseContinuationCallback promiseContinuationCallback = delegate (JavaScriptValue task, IntPtr callbackState)
             {
-                taskQueue.Enqueue(task);
+               // taskQueue.Enqueue(task);
```

Comment:
if we decide to remove the taskQueue, let's just remove these and not just comment out. I wonder if this queue gets populated ever. Can you check?